### ISSUE TITLE
False positive: curl used in place of get_url module

### ISSUE THREAD

**Issue by xeraa** [willthames/ansible-lint#70]
```
$ ansible-lint --version
ansible-lint 2.1.0
```

We need to do a POST (not a GET) with cURL, but ansible-lint throws an `[ANSIBLE0006] curl used in place of get_url module`

Example (not using the Slack module, since it doesn't support attachments ATM):

```
command: >
  curl -XPOST  -d '{{ slack_body | to_json }}' https://hooks.slack.com/services/{{slack.token}}
```

I would assume the check needs to make sure it's actually a cURL GET to have a match.

**Comment by shaleh** [comment]
The ansible uri module documentation has this example:

'''
# Create a JIRA issue
- uri:
    url: https://your.jira.example.com/rest/api/2/issue/
    method: POST
    user: your_username
    password: your_pass
    body: "{{ lookup('file','issue.json') }}"
    force_basic_auth: yes
    status_code: 201
    body_format: json
'''

Surely you could do the same?

**Comment by xeraa** [comment]
My bad, that's true.

Maybe the output could say "curl used in place of the get_url or uri module", which could avoid some confusion :)

**Comment by willthames** [comment]
Thanks @xeraa and @shaleh for your contributions on this one, happy to have a quick fix on this one!
